python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -76,6 +87,10 @@ def _get_dbt_target_path() -> Path:
     return Path(os.getenv("DBT_TARGET_PATH", "target"))
 
 
+# TODO: Need to figure out a better place for this to live so it isn't globally scoped.
```

Comment:
heavy plus one. if I understand correctly, the ideal scope is within a single step execution? and so the question is how to tie it to that?

Reply:
I thought a little more about this and wonder if we could basically have a cache that lives on the `DbtCliResource` instance?

Reply:
We could also make the user use `dbt compile` when creating their manifest. That way, their manifest is populated with `compiled_code`, which we could use to trace the lineage.

This also addresses the issue from above, where subsetting may not include all the compiled models.

Reply:
aren't we already going to need some other way of getting the upstream column metadata, for sources and seeds?